package-lock.json

```diff
@@ -8245,7 +8245,7 @@
         },
         "p-cancelable": {
           "version": "0.4.1",
-          "resolved": "https://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
```

Comment:
We usually avoid pushing up changes to `package-lock.json` except in the case where `mattermost-redux` needs to be updated. Can we revert the changes that aren't to the `mattermost-redux` repo?

Reply:
Yeah sorry, included this for testing only